Docs/Configuration.tex

```diff
@@ -7560,6 +7560,37 @@ \subsection{Properties}\label{uefiprops}
   could be the second 256 MB corrupted by the Intel HD 3000 or an area with faulty RAM.
   Refer to the \hyperref[uefirsvdprops]{ReservedMemory Properties} section below for details.
 
+\item
+  \texttt{Unload}\\
+  \textbf{Type}: \texttt{plist\ array}\\
+  \textbf{Failsafe}: Empty\\
+  \textbf{Description}: Unload specified firmware drivers.
+
+  To be filled with \texttt{plist\ string} entries containing
+  the names of firmware drivers to unload before loading the
+  \texttt{Drivers} section.
+  This setting is typically only required if a user-provided driver is a variant of
+  an existing system firmware driver, and if the new driver would detect itself
+  as partially loaded, or otherwise fail to operate correctly, if the old
+  driver is not unloaded first.
+
+  \textbf{Warning}: Unloading system firmware drivers is usually not required
```

Comment:
Maybe drop a sentence about buggy drivers? Iirc OpenUsbKbDxe be used on Aptio IV because it’s USB drivers have borked unload.

Reply:
> ...OpenUsbKbDxe be used..

cannot be used? must be used?

I gather we cannot unload OpenUsbKbDxe on that firmware... but then... we have to/can/must...?

Reply:
Cannot, whoops. No, the driver we want to replace with OpenUsbKbDxe cannot be unloaded. And no, point is unloading may be very broken and it depends on the driver unloaded whether it works.

Reply:
So just strengthening this warning. Okay.

Reply:
> Cannot, whoops. No, the driver we want to replace with OpenUsbKbDxe cannot be unloaded. And no, point is unloading may be very broken and it depends on the driver unloaded whether it works.

Btw I already found some drivers relevant to network boot on Apple which at least refuse to unload with Unsupported, which is better than borking everything. So i.e. it can be different levels of broken, but yes, certainly can be completely broken. (I was initially trying to avoid this on the basis that it probably wouldn't work half the time - before, see other comment, deciding it probably was out best bet.)